common/lib/common-utils/src/bufferNode.ts

```diff
@@ -37,3 +37,11 @@ export function Uint8ArrayToString(arr: Uint8Array, encoding?: string): string {
         return Buffer.from(arr).toString(encoding);
     }
 }
+
+/**
+ * Convert base64 or utf8 string to array buffer
+ */
+export function toBuffer(input: string, encoding: string): ArrayBufferLike {
+    const iso = IsoBuffer.from(input, encoding);
+    return iso.buffer.slice(iso.byteOffset, iso.byteOffset + iso.byteLength);
```

Comment:
I think iso.buffer should work here too. If it does not - I'm curious why? Slicing will create more memory allocations...
Will this be better:

        // In a Node environment, IsoBuffer may be a Node.js Buffer.  Node.js will
        // pool multiple small Buffer instances into a single ArrayBuffer, in which
        // case we need to slice the appropriate span of bytes.
        return iso.byteLength === iso.buffer.byteLength
            ? iso.buffer
            : iso.buffer.slice(iso.byteOffset, iso.byteOffset + iso.byteLength);